pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -1430,12 +1431,56 @@ type CephObjectStoreUserList struct {
 
 // ObjectStoreUserSpec represent the spec of an Objectstoreuser
 type ObjectStoreUserSpec struct {
-	//The store the user will be created in
+	// The store the user will be created in
 	// +optional
 	Store string `json:"store,omitempty"`
-	//The display name for the ceph users
+	// The display name for the ceph users
 	// +optional
 	DisplayName string `json:"displayName,omitempty"`
+	// +optional
+	// +nullable
+	Capabilities *CapSpec `json:"capabilities,omitempty"`
+	// +optional
+	// +nullable
+	Quotas *RGWUserQuotaSpec `json:"quotas,omitempty"`
+}
+
+//The additional capabilities for the ceph user
+type CapSpec struct {
+	// This gives permission to create/modify/delete users/subusers/keys.
+	// +optional
+	// +kubebuilder:validation:Enum=*;read;write
```

Comment:
I've made this same comment 3 times now: what about the `read, write` capability documented in the Ceph docs here? https://docs.ceph.com/en/latest/radosgw/admin/?#add-remove-admin-capabilities

Reply:
Fourth time. What about the `read, write` capability?

Reply:
> --caps="[users|buckets|metadata|usage|zone]=[*|read|write|read, write]"

Reply:
@BlaineEXE Sorry I am not getting it, Do I need to add
`The read, write or all capabilities to users, buckets, metadata and usage (utilization)` on the top CapSpec instead of individual fields
Thanks for being patient with me.

Reply:
The specification from Ceph says that there are four valid enums for the caps, not 3. 
- `*`
- `read`
- `write`
- `read, write`

Reply:
@BlaineEXE : even if the user sets `read, write` at the backend the value is `*` So atm I don't felt any much difference by adding the fourth value. And I am not sure how it will work as enum. Do `read, write` treat as separate enum
?

Reply:
Sorry it was not clear for me earlier, I thought of explaining about read/write in the `CapSpec`

Reply:
We direct the user to the Ceph documentation, so we should support all the values that Ceph does, even if `read, write` is the same as `*`. If a user tries to use `read, write`, they will get an error even though it should be supported.